provider/src/main/java/com/cloudera/director/google/compute/GoogleComputeInstanceTemplateConfigurationProperty.java

```diff
@@ -63,6 +63,33 @@
       .required(false)
       .build()),
 
+  NETWORK_PROJECT(new SimpleConfigurationPropertyBuilder()
+      .configKey("networkProject")
+      .name("Network Project")
+      .defaultDescription(
+          "The project the nework belongs to.<br />" +
+              "<a target='_blank' href='https://cloud.google.com/compute/docs/networking#networks'>More Information</a>")
+      .defaultValue(null)
+      .required(false)
+      .build()),
+
+  ASSIGN_EXTERNAL_IPS(new SimpleConfigurationPropertyBuilder()
+      .configKey("assignExternalIPs")
+      .name("Assign External IPs")
+      .defaultDescription("Assign external IP addresses to created instances.  Cloudera director will" +
```

Comment:
Cloudera Altus Director

Also, s/will still require/may require - it's possible to point Director to package repository mirrors within a private network, when outbound access is prohibited.